### ISSUE TITLE
Pressing enter does not continue docblock

### ISSUE THREAD

**Issue by ianmaddox** [jeremyvii/vs-docblockr#94]
**Describe the bug**
Pressing enter inside a docblock does not continue the docblock. The asterisk is not printed and the cursor moves to the beginning of the line as if it is outside a docblock.

**To Reproduce**

1. Start a docblock with /** and press Enter
2. The cursor is now inside a docblock template. Type some text and press Enter
3. Notice that the cursor has moved to the beginning of the line and no asterisk was created

![docblock not continuing](https://user-images.githubusercontent.com/1526438/232279994-38abe647-9a48-4c4c-b001-2b4a68e287d2.gif)

**Expected behavior**
A newline and "  * " is printed with my cursor at the end of the newly created line.

**Code Snippet**
```
/**
  * I'm going to press enter>
Oops, I got a new line with no padded asterisk
  */

```

**Enviroment (please complete the following information):**
 - OS: Windows
 - Version 11
 - Language(s) JavaScript

**Comment by jeremyvii** [comment]
I am not able to replicate this issue, although I am not Ubuntu rather than Windows. It may be a bit before I can test on a Windows machine. 

Do you have any other extensions installed?

**Comment by ianmaddox** [comment]
Yes, though I recently went through the process of disabling every plugin and re-enabling them one by one. I wasn't testing for this problem in particular, but I did notice that the problem persisted even with everything turned off including vs-dockblockr and then went away once I'd enabled some unknown combination of extensions. Before I embark on another round of extension disabling, I should check if the problem is configuration. I have no idea how to check the DB continuation string setting to confirm this theory. Do you know where I should look?
